### PR TITLE
Update col plan vignette #414

### PR DESCRIPTION
**Pull request by karima662**
**addressing issue #414**

- added example of renaming spanners (within same level & on different levels)
- added depiction of table for first `dat` data at the beginning to highlight the changes made when deselecting a column in the next section
- renamed the section for "naming the group-label header"
- added a little more information to some code explanations
- added comments to some code chunks to highlight the specific line/s of code being added (especially since a lot of code is repeated for each example, which can make it difficult for beginners to decipher what has changed in the code) - feel free to a share a counter opinion

### CODE REVIEW

File: vignettes/col_plan.Rmd [hunk 1]
```diff
@@ -192,6 +211,33 @@ tfrmt(
...
+  param = parm,
+  value = val,
+  column = c(span2, span1, my_col),
+  body_plan = body_plan(
+    frmt_structure(group_val = ".default", label_val = ".default", frmt("x"))
+  ),
+  col_plan = col_plan(
+    # rename column spanner in same level
+    span_structure(span1 = c("first cols" = "cols 1,2"),
+                   span1 = c("just col4" = "col 4")),
```
Comment (bzkrouse) [inline vignettes/col_plan.Rmd:229]: these should be combined into a single named vector:
````
span_structure(span1 = c("first cols" = "cols 1,2",
                                          "just col4" = "col 4"))
```

**Review by bzkrouse — CHANGES_REQUESTED** [review]
Thanks for the really helpful edits for clarification!! just 1 small requested change :)

File: vignettes/col_plan.Rmd (hunk 1 above)
Comment (karima662) [reply vignettes/col_plan.Rmd]: addressed here: https://github.com/GSK-Biostatistics/tfrmt/pull/420/commits/0472a0c8453ced9154a8523557af69513b7822fe

please resolve conversation if new commit meets requirements :)

**Review by bzkrouse — APPROVED** [review]
Looks great thank you!

**Pull request merged**